components/suggestion/search_suggestion_list.tsx

```diff
@@ -8,14 +8,48 @@ import Constants from 'utils/constants';
 
 import Popover from 'components/widgets/popover';
 
-import SuggestionList from './suggestion_list.jsx';
+import SuggestionList from './suggestion_list';
+import {UserProfile} from './command_provider/app_command_parser/app_command_parser_dependencies';
 
-export default class SearchSuggestionList extends SuggestionList {
-    static propTypes = {
-        ...SuggestionList.propTypes,
+interface Item extends UserProfile {
```

Comment:
The item can actually be more different types than just a `UserProfile`. Those fields you're adding come from when it's a `Channel`, and outside of the search box, it can also be many other things like a slash command or emoji. I think the old way of having a named version of `any` is good enough here since this code wasn't written in a way that makes typing it easy.

Reply:
Thank you for the heads up.